frontend/src/components/LogIn/login.css

```diff
@@ -0,0 +1,120 @@
+body {
+    background-color: #F4EFFA;
+    padding-top: 10%;
+    padding-bottom: 10%;
+}
+
+h1 {
+    letter-spacing: 5px;
+    margin-bottom: 5%;
+}
+
+.loginGroup {
+    font-weight: 700;
+    text-align:center;
+    color: #2F184B;
+
+    border-radius: 25px;
+    padding-top: 5%;
+    padding-bottom: 5%;
+    margin: auto;
+    max-width: 30%;
+    max-height: 20%;
+    background-color: rgb(253, 250, 255);
+    box-shadow: 0px 0px 6px #2F184B;
+}
+
+label {
+    background-color: rgb(253, 250, 255); 
+    width: auto;
+    display:block;
+    text-align:left;
+    padding-left: 20%;
+    margin-bottom: 1%;
+}
+
+input {
+    width: 60%;
+    height: 80%;
+    font-size: 20px;
+    border: none;
+    box-shadow: 0px 0px 3px #2F184B;
+    border-radius: 4px;
+    margin-bottom: 3%;
+}
+
+.passReset{
```

Comment:
I'm noticing some small inconsistencies on the white space in this file

i.e. `input {` vs `passReset{` (note the white space). Lets be consistent in our style and add a white space after the selector for entire file. `passRest {`